packages/trader/src/AppV2/Components/DraggableList/draggable-list-item.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { StandaloneGripDotsVerticalBoldIcon, StandaloneCircleMinusFillIcon } from '@deriv/quill-icons';
+
+type DraggableListItemProps = {
+    title: string;
+    leftIcon?: React.ReactNode;
+    rightIcon?: React.ReactNode;
+    onLeftIconClick?: () => void;
+    onRightIconClick?: () => void;
+}
+
+const DraggableListItem: React.FC<DraggableListItemProps> = ({
+    title,
+    leftIcon,
+    rightIcon,
+    onLeftIconClick,
+    onRightIconClick,
+}) => {
+    const default_left_icon = <StandaloneGripDotsVerticalBoldIcon iconSize='sm' />;
+    const default_right_icon = <StandaloneCircleMinusFillIcon fill='var(--core-color-solid-red-700)' iconSize='sm' />;
```

Comment:
It might be better to move `fill` to css, but as icon was created with the ability to accept it, then it's up to you

Reply:
I'd say since the icons accept the fill option we are better passing it. I don't see any advantage of adding it through css)